### PR TITLE
feat - responsive navbar

### PR DESCRIPTION
**Pull request by biswanath-tewari**

### CODE REVIEW

File: components/navbar.css [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  display: flex;
+  justify-content: space-between;
+  width: 30%;
+}
+
+.nav-links li {
+  list-style: none;
+}
+
+.nav-links > li > a {
```
Comment (yash9263) [inline components/navbar.css]: can give `<a>` tag a name like `nav-link` instead of nesting two levels down

**Comment by yash9263** [comment]
looks good to me

**Review by Bibin-Antony — APPROVED** [review]

**Review by yash9263 — APPROVED** [review]

File: components/navbar.css [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+  .burger i {
+    font-size: 1.5rem;
+  }
+
+  .navlinks-active {
+    transform: translateX(0);
+  }
+
+  .nav-links li {
```
Comment (Bibin-Antony) [inline components/navbar.css:81]: you could add a hover effect on the nav labels. For home Docs and the rest.

**Review by Bibin-Antony — CHANGES_REQUESTED** [review]

**Review by SukanyaSen530 — APPROVED** [review]
Looks Good to be merged

File: components/navbar.css (hunk 1 above)
Comment (biswanath-tewari) [reply components/navbar.css]: refactored

File: components/navbar.css (hunk 2 above)
Comment (biswanath-tewari) [reply components/navbar.css:81]: it is already there, as you can see in the preview. Thanks for input.

**Comment by sandeepkumardev-zz** [comment]
all good 👍

**Pull request merged**